installer/conf/rsyslog-lad.conf

```diff
@@ -1,2 +1,2 @@
 # OMS Syslog collection for workspace LAD (SYSLOG_PORT=%SYSLOG_PORT%)
-# To be added for each facility/min_severity: <facility>.<min_severity>     @127.0.0.1:%SYSLOG_PORT%
```

Comment:
@canfikret I know this change migh not cause any conflicts with LAD, but just asking if any issue may happend ?